meorg_client/tests/test_cli.py

```diff
@@ -48,6 +48,15 @@ def test_file_multiple(runner):
     time.sleep(5)
 
 
+def test_file_upload_parallel(runner):
+    """Test file-upload via CLI."""
+
+    # Upload a tiny test file
+    filepath = os.path.join(mu.get_installed_data_root(), "test/test.txt")
```

Comment:
A fixture returning the filepath `test/test.txt` would be good here

Reply:
Hmmm, I thought I added a function for that. Good point! Will fix.